squire/urls.py

```diff
@@ -1,22 +1,13 @@
-"""base URL Configuration
-
-The `urlpatterns` list routes URLs to views. For more information please see:
-    https://docs.djangoproject.com/en/2.2/topics/http/urls/
-Examples:
-Function views
-    1. Add an import:  from my_app import views
-    2. Add a URL to urlpatterns:  path('', views.home, name='home')
-Class-based views
-    1. Add an import:  from other_app.views import Home
-    2. Add a URL to urlpatterns:  path('', Home.as_view(), name='home')
-Including another URLconf
-    1. Import the include() function: from django.urls import include, path
-    2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
-"""
 from django.contrib import admin
 from django.urls import path, include
 
 urlpatterns = [
+    # Admin Panel
     path('admin/', admin.site.urls),
-    path('calendar/', include('activity_calendar.urls')),
+    # Activity Calendar
+    path('calendar/', include('activity_calendar.urls-frontend')),
+    path('api/calendar/', include('activity_calendar.urls-api')),
+    # Membership File
+    path('members/', include('membership_file.urls_frontend')),
+    path('api/members/', include('membership_file.urls_api')),
```

Comment:
Why are we even creating an API? Are there interesting consumers at this point?

Reply:
It'll be useful later on, as members should be able to update their own information (without the need for the Django admin panel). The forms will need a location to send their POST/PUT data to.

Reply:
That's usually not done using an API, but using a POST method that accepts HTTP forms.